x-pack/packetbeat/cmd/root.go

```diff
@@ -11,6 +11,7 @@ import (
 	_ "github.com/elastic/beats/v7/x-pack/libbeat/include"
 
 	// This registers the Npcap installer on Windows.
+	"github.com/elastic/beats/v7/x-pack/libbeat/management"
```

Comment:
Please put this in a separate stanza. The addition here makes the comment incorrect.